nautobot_chatops/templates/nautobot/access_grant_edit.html

```diff
@@ -2,7 +2,7 @@
 {% load static %}
 {% load form_helpers %}
 
-<h1>{% block title %}{% if obj.pk %}Editing{% else %}Add a new{% endif %} Nautobot Access Grant{% endblock %}</h1>
+<h1>{% block title %}{% if obj.command %}Editing{% else %}Add a new{% endif %} Nautobot Access Grant{% endblock %}</h1>
```

Comment:
IIRC this and the other below can be changed to `if obj.present_in_database`.

Reply:
Yup that worked.  I changed both.  Thank you.